assets/config/postcss.config.js

```diff
@@ -16,6 +16,18 @@ module.exports = {
             ]
         }),
 
+        // PurgeCSS to remove unused classes for better page speed.
+        require("@fullhuman/postcss-purgecss")({
+            // Specify the paths to all of the template files in your project
+            content: [ "./layouts/**/*.html" ],
```

Comment:
We also use selectors in some of our scripts, such as `search.js` to add HTML dynamically, and I think a couple other scripts. Might be worth just including them:

```javascript
// Some of our scripts reference CSS classes.
"./assets/js/**/*.js",
```